islandora.api.php

```diff
@@ -620,7 +626,7 @@ function hook_islandora_datastream_access($op, $object, $user) {
  *
  * @see hook_islandora_datastream_access()
  */
-function hook_cmodel_pid_islandora_datastream_access($op, $object, $user) {
+function hook_cmodel_pid_islandora_datastream_access($op, $datastream, $user) {
```

Comment:
Why is the signature here uncasted, means different for $datastream than the no cmodel specific hook?